.github/workflows/bindings_java.yml

```diff
@@ -90,6 +90,6 @@ jobs:
           ./mvnw test -Dtest="behavior.*Test"
 
           export OPENDAL_TEST=fs
-          export OPENDAL_FS_ROOT=/tmp
+          export OPENDAL_FS_ROOT=${{ runner.temp }}/
```

Comment:
"Behavior tests" should be removed. We have already have them in our new test framework.

Reply:
Do we? Where we test for FS and memory backend?

Reply:
memory is missing, we can add them via a new PR which should be simple.